src/components/users/views.tsx

```diff
@@ -90,7 +90,7 @@ export function UserPage(props: IUserPageProps): ReactElement {
         <div className="govuk-summary-list__row">
           <dt className="govuk-summary-list__key">Last logon</dt>
           <dd className="govuk-summary-list__value">
-            {format(new Date(props.lastLogon), DATE_TIME)}
+            {props.lastLogon && format(new Date(props.lastLogon), DATE_TIME)}
```

Comment:
I was kinda sorta hoping it would come through as e.g. `undefined` if that's what it is, but I might have to do some `toString` shenanigans for that to work come to think...